recipes-bsp/u-boot/u-boot_2015.10.bbappend

```diff
@@ -0,0 +1,14 @@
+# Keep this separately from the rest of the .bb file in case that .bb file is
+# overridden from another layer.
+
+require u-boot-mender.inc
+
+# Copy script to the deploy area with u-boot, uImage, and rootfs
+do_deploy_uenv () {
+    if [ -e ${WORKDIR}/uEnv.txt ] ; then
+        install -d ${DEPLOY_DIR_IMAGE}
+        install -m 0444 ${WORKDIR}/uEnv.txt ${DEPLOY_DIR_IMAGE}
```

Comment:
This doesn't look right to me, it will put the `uEnv.txt` file in `tmp/deploy/images/vexpress-qemu`, together with the partitioned image, but not *inside* the image. Is that really what you want?